src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const getPartnerLabel = () => {
+    let marriedLabel;
+
+    if (isMarried) {
+      marriedLabel = sex === 'm'
+        ? `${partnerName} is my wife`
+        : `${partnerName} is my husband`;
+    }
+
+    return isMarried === true
```

Comment:
```suggestion
    return isMarried
```
isMarried is a boolean value, so no need to compare it with `true`, moreover in the ternary the variables in the condition part will be automatically transformed into boolean (the same as in the `if` block condition) - FYI